src/azul/__init__.py

```diff
@@ -1590,6 +1590,26 @@ def docker_image_gists_path(self) -> Path:
 
     waf_rate_rule_limit = 1000
 
+    @property
+    def waf_file_download_limit(self) -> tuple[int, int, int] | None:
+        """
+        :return: a triple of
+            1. The rate limit
+            2. The evaluation window
+            3. The Retry-After header value
+        """
+        values = self.environ.get('AZUL_FILE_DOWNLOAD_RATE_LIMIT')
+        if values is None:
+            return None
+        else:
+            rate, values = values.split('/')
+            window, concurrency = values.split('@')
+            rate = int(rate)
+            window = int(window)
+            concurrency = float(concurrency)
+            retry_after = round(window / rate * concurrency)
+            return rate, window, retry_after
```

Comment:
I changed my mind. This should return the rate, so that we can separate the parsing code from the computation of the retry-after header value. The latter can go into the TF config.

Reply:
I'm not sure I entirely understand, as this seems to contradict [your previous review comment](https://github.com/DataBiosphere/azul/pull/6749#discussion_r1887334582) saying to return an `attrs` class.

Reply:
Yes, return the rate as part of the instance of that class.